src/main/java/dev/meyi/bn/modules/CraftingModule.java

```diff
@@ -1,16 +1,20 @@
 package dev.meyi.bn.modules;
 
+import com.google.gson.JsonIOException;
 import dev.meyi.bn.BazaarNotifier;
 import dev.meyi.bn.config.ModuleConfig;
+import dev.meyi.bn.modules.calc.CraftingCalculator;
 import dev.meyi.bn.utilities.ColorUtils;
 import dev.meyi.bn.utilities.Defaults;
-import dev.meyi.bn.modules.calc.CraftingCalculator;
-import java.awt.Color;
+import dev.meyi.bn.utilities.Utils;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.gui.Gui;
+
+import java.awt.*;

```

Comment:
Is this even used? Either get rid of the * import and expand it out or remove the line entirely